apps/web/src/common/hoc/withGuard/index.tsx

```diff
@@ -18,7 +19,7 @@ const withGuard = (
       return null
     }
 
-    if (!user.isVerified) {
+    if (user.status !== UserStatus.ACCEPTED) {
```

Comment:
Should this have 2 cases ?
- REJECTED
- PENDING

Reply:
will be fixed when reject page is implemented